src/main/java/io/appium/java_client/HasSessionDetails.java

```diff
@@ -17,15 +17,20 @@
 package io.appium.java_client;
 
 import static io.appium.java_client.MobileCommand.GET_SESSION;
+import static io.appium.java_client.MobileCommand.GET_ALLSESSION;
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.toMap;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 
 import org.openqa.selenium.remote.Response;
 
+import java.util.Arrays;
```

Comment:
Looks like an unused import.

Reply:
yes, removed that line @SrinivasanTarget